src/admin/dumb_components/custom_email_form.js

```diff
@@ -1,19 +1,22 @@
 const React = require('react')
-const { map, objOf, zipWith, merge, compose } = require('ramda')
+const { map, objOf, zipWith, merge, compose, indexBy, prop, converge, dissoc } = require('ramda')
 
 export default ({ members, submit }) => {
   const onSubmit = (e) => {
     e.preventDefault()
-    const template = format_message(e.target[0].value)
-    const emailBodies = map(compose(objOf('email_body'), template), members)
-    const emails = zipWith(merge, members, emailBodies)
-    submit(emails)
+    const template = format_message(e.target)
+    const emailBodies = map(compose(objOf('content'), template), members)
+    const emailsArr = zipWith(merge, members, emailBodies)
+    const setEmailKey = map(compose(indexBy, prop), ['primary_email', 'secondary_email'])
```

Comment:
would it be ok to shape the payload into this in the reducer, and save it to state:

```
{ 'jmurphy@gmail': { content: [] }
, 'ivangon@gmail': { content: [] }
}
```

Upon entering a custom email body, the custom email component maps through the above state injecting the content array, then calls the "submit_email" action which heads to backend and sends the email.

It would still leave some functionality to the component, but hopefully it is not too much